src/services/auth-services.ts

```diff
@@ -54,7 +54,14 @@ export default class AuthServices {
           folioNumber: response.data.folioNumber
         }
       }
-      throw new Error('Invalid response data ')
+      throw new Error('Invalid response data')
+    }).catch(error => {
+      if (error?.response?.status === StatusCodes.FORBIDDEN) {
+        return { status: 'FORBIDDEN' } as AuthInformationIF
+      } else if (error?.response?.status === StatusCodes.NOT_FOUND) {
+        return { status: 'NOT_FOUND' } as AuthInformationIF
+      }
+      throw error
```

Comment:
This code checks for an exact response status (eg, 403). I decided to abstract the `status` given back to the caller, since the caller shouldn't know about network codes. So it's a string, which is OK for just 2 values (otherwise I'd create an enum).

This code specifically handles 403 (forbidden) and 404 (not found). Any other error is thrown as a new exception, which our caller will turn into a null object.